src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -41,11 +43,30 @@
    * @param symbols list of symbols to get last traded price for.
    * @return a List of IexLastTradedPrice objects for the given symbols.
    */
+
   @GetMapping(value = "${mvc.iex.getLastTradedPricePath}", produces = {
       MediaType.APPLICATION_JSON_VALUE})
   public List<IexLastTradedPrice> getLastTradedPrice(
       @RequestParam(value = "symbols") final List<String> symbols) {
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get Historical Prices
+   *
```

Comment:
Javadoc comments aren't always required, but they should be filled out if they exist